Gemfile

```diff
@@ -19,6 +19,8 @@ group :integration do
   gem "kitchen-dokken"
   gem "kitchen-inspec"
   gem "kitchen-vagrant"
+  gem "inspec", "~> 5.22" # Fix dependency conflicts for ruby < 3.4
+  gem "train", "~> 3.10" # Fix dependency conflicts for ruby < 3.4
```

Comment:
Should we make this the last <3.4 release?

Reply:
that'd make sense especially after chef-client 19 gets released as it'll also be based on ruby 3.4.